repository/src/main/java/org/apache/atlas/repository/store/graph/v2/preprocessor/AuthPolicyValidator.java

```diff
@@ -267,7 +271,27 @@ public void validate(AtlasEntity policy, AtlasEntity existingPolicy,
         }
     }
 
-    private static void validateParentUpdate(AtlasEntity policy, AtlasEntity existingPolicy) throws AtlasBaseException {
+    private void validateEntityResources(String connQn, List<String> resources) throws AtlasBaseException {
+        List<String> entityResources = getFilteredPolicyResources(resources, RESOURCES_ENTITY);
```

Comment:
does value of resource entity include `:`

Reply:
Yes